reduxApp2/dev/js/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+import {combineReducers} from 'redux';
+import UserReducer from './reducer-users';
+ 
+
+const allReducers = combineReducers({
```

Comment:
@VARTIKASINGH1234 can we add the reducer without using 'combineReducers' ? because here we're using only one reducer.

Reply:
yes, we can do this, but I have made this, because in case in  future , we may need to add more reducers